sycl/test/basic_tests/stream.cpp

```diff
@@ -155,6 +155,9 @@ int main() {
         Out << ConstPtr << sm::endl;
         auto multiPtr = private_ptr<int>(Ptr);
         Out << multiPtr << sm::endl;
+// CHECK-NEXT: 0x{{[0-9a-fA-F]*$}}
+// CHECK-NEXT: 0x{{[0-9a-fA-F]*$}}
+// CHECK-NEXT: 0x{{[0-9a-fA-F]*$}}
```

Comment:
Do you I get it correctly here we might print `545`?

It would be nice if you describe the problem in the log message.

Reply:
You are right, the problem is that printed pointer value could contain "545". That will cause a failure.

Reply:
Described the problem in the log message.